cirq-google/cirq_google/engine/virtual_engine_factory.py

```diff
@@ -84,6 +91,29 @@ def _create_perfect_calibration(device: cirq.Device) -> calibration.Calibration:
     return calibration.Calibration(calibration=snapshot, metrics=all_metrics)
 
 
+def load_median_device_calibration(processor_id: str) -> calibration.Calibration:
+    """Loads a median Calibration for the given device.
+
+    Real calibration data from Google's 'rainbow' and 'weber' devices has been
+    saved in Cirq. The calibrations selected are roughly representative of the
+    median performance for that chip.
+
+    Args:
+         processor_id: name of the processor to simulate.
+
+    Raises:
+        ValueError: if processor_id is not a supported QCS processor.
+    """
+    cal_name = MEDIAN_CALIBRATIONS.get(processor_id, None)
+    if cal_name is None:
+        raise ValueError(f"Got processor_id={processor_id}, but no such processor is defined.")
```

Comment:
Maybe mention median calibration in here, so it is clear where the value error is coming from.

Reply:
Done.